hyper/hyper/hyper.go

```diff
@@ -10,25 +10,38 @@ import (
 
 // Interface is an interface with all the actions that a hypervisor has to be able to do
 type Interface interface {
-	// Get returns a desktop using it's name
+	// Get returns a running desktop object from it's name
```

Comment:
It's not an object, it's a struct. But it doesn't matter, it's already described in the return arguments

Reply:
ok